hs_core/management/commands/migrate_relation_types.py

```diff
@@ -0,0 +1,120 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+
+from hs_core.hydroshare import utils
+from hs_core.hydroshare.utils import set_dirty_bag_flag
+from hs_core.models import BaseResource
+
+
+class Command(BaseCommand):
+    help = "Migrate source metadata to relation metadata, and update relation types and values of all resources."
+
+    def handle(self, *args, **options):
+        log = logging.getLogger()
+        res_count = BaseResource.objects.count()
+        log_msg = "Total resources:{}".format(res_count)
+        print(log_msg)
+        log.info(log_msg)
```

Comment:
Do we really need to put info messages in log after printing them out in the console when running the command?

Reply:
I would prefer to log what relation types are migrated for each of the resources for future reference. I can see line 19 is probably not very useful for logging. Will remove that line.